lottery.py

```diff
@@ -22,27 +22,28 @@
 import teams
 import repositories
 import lottery_modes
+import database_helper
 
 class Lottery(object):
     def __init__(self):
-        self._reviewers = {}
         self._reviewers_by_teams = {}
         self._ubers_by_teams = {}
         self._reviewer_selector = None
 
     def select_assignee(self, issue, team_name):
-        selected = self._reviewer_selector(dict(map(lambda r: (r, self._reviewers[r] if r in self._reviewers else 0),
+        selected = self._reviewer_selector(dict(map(lambda x: (x, database_helper.get_user_score(x)),
                                                     self._reviewers_by_teams[team_name])),
                                            self._ubers_by_teams[team_name], issue)
         if selected is None:
             selected = issue.author
         issue.assignee = selected
-        self._reviewers[issue.assignee] += 1
+        database_helper.update_user_rating(selected)
+
 
     def increase_reviewer_score(self, reviewer):
-        if reviewer not in self._reviewers:
+        if reviewer not in self._reviewers_by_teams:
```

Comment:
It shouldn't work, should it? It will just check if there is a team named the same way as reviewer you want to increment